client/components/recycleAndReuseComponents/AddItemMultiSelect.js

```diff
@@ -0,0 +1,75 @@
+import { Button, Flex } from '@chakra-ui/react'
+import React from 'react'
+import Select from 'react-select'
+
+const AddItemMultiSelect = ({ data, setItems, willTriggerDialog }) => {
```

Comment:
Refactored `MultiSelect` to `AddItemMultiSelect` to make it clear that this is only for `AddItem` component